### ISSUE TITLE
Lacking the option for 

### ISSUE THREAD

**Issue by ldigas** [reedes/vim-thematic#16]
My guifont is 
    guifont=Courier_New:h9:cEASTEUROPE  

which in vim-thematic comes down to

    let g:thematic#themes = {
    \'badwolf' : {'background' : 'dark',
    \             'colorscheme' : 'badwolf',
    \             'typeface' : 'Consolas',
    \             'font-size' : 9,
    \             }
    \}

Is vim-thematic missing the option to set encoding?

**Comment by reedes** [comment]
Yep, Thematic is currently missing a 'font-encoding' feature, but that can be added.

I'm not sure how soon I can get to it, but a pull-request with the change from you or another user would be welcome.
